ruleset/rules/0800-sysmon_id_1.xml

```diff
@@ -250,4 +250,40 @@
mage":"C:\\\\Windows\\\\System32\\\\WindowsPowerShell\\\\v1.0\\\\powershell.exe","product":"Microsoft® Windows® Operating System","parentProcessGuid":"{4dc16835-6d03-60f8-3101-000000005300}","description":"Windows PowerShell","logonGuid":"{4dc16835-577d-60f8-7c02-1b0000000000}","parentCommandLine":"ShadowSteal.exe","processGuid":"{4dc16835-6d03-60f8-fd83-590000000000}","logonId":"0x1b027c","parentProcessId":"4832","processId":"4636","currentDirectory":"C:\\\\Users\\\\AtomicRed\\\\Downloads\\\\shadow2\\\\","utcTime":"2021-07-21 18:52:51.728","hashes":"SHA1=F43D9BB316E30AE1A3494AC5B0624F6BEA1BF054,MD5=04029E121A0CFA5991749937DD22A1D9,SHA256=9F914D42706FE215501044ACD85A32D58AAEF1419D404FDDFA5D3B48F66CCD9F,IMPHASH=7C955A0ABC747F57CCC4324480737EF7","parentImage":"C:\\\\Users\\\\AtomicRed\\\\Downloads\\\\shadow2\\\\ShadowSteal.exe","ruleName":"technique_id=T1086,technique_name=PowerShell","company":"Microsoft Corporation","commandLine":"powershell.exe -c \\\"[System.IO.File]::Copy('\\\\\\\\?\\\\GLOBALROOT\\\\Device\\\\HarddiskVolumeShadowCopy1\\\\Windows\\\\System32\\\\config\\\\SECURITY', 'tmp\\\\202107211152_SECURITY')\\\"","integrityLevel":"High","fileVersion":"10.0.19041.546 (WinBuild.160101.0800)","user":"EXCHANGETEST\\\\AtomicRed","terminalSessionId":"1"},"system":{"eventID":"1","keywords":"0x8000000000000000","providerGuid":"{5770385f-c22a-43e0-bf4c-06f5698ffbd9}","level":"4","channel":"Microsoft-Windows-Sysmon/Operational","opcode":"0","message":"\"Process Create:\r\nRuleName: technique_id=T1086,technique_name=PowerShell\r\nUtcTime: 2021-07-21 18:52:51.728\r\nProcessGuid: {4dc16835-6d03-60f8-fd83-590000000000}\r\nProcessId: 4636\r\nImage: C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\powershell.exe\r\nFileVersion: 10.0.19041.546 (WinBuild.160101.0800)\r\nDescription: Windows PowerShell\r\nProduct: Microsoft® Windows® Operating System\r\nCompany: Microsoft Corporation\r\nOriginalFileName: PowerShell.EXE\r\nCommandLine: powershell.exe -c \"[System.IO.File]::Copy('\\\\?\\GLOBALROOT\\Device\\HarddiskVolumeShadowCopy1\\Windows\\System32\\config\\SECURITY', 'tmp\\202107211152_SECURITY')\"\r\nCurrentDirectory: C:\\Users\\AtomicRed\\Downloads\\shadow2\\\r\nUser: EXCHANGETEST\\AtomicRed\r\nLogonGuid: {4dc16835-577d-60f8-7c02-1b0000000000}\r\nLogonId: 0x1B027C\r\nTerminalSessionId: 1\r\nIntegrityLevel: High\r\nHashes: SHA1=F43D9BB316E30AE1A3494AC5B0624F6BEA1BF054,MD5=04029E121A0CFA5991749937DD22A1D9,SHA256=9F914D42706FE215501044ACD85A32D58AAEF1419D404FDDFA5D3B48F66CCD9F,IMPHASH=7C955A0ABC747F57CCC4324480737EF7\r\nParentProcessGuid: {4dc16835-6d03-60f8-3101-000000005300}\r\nParentProcessId: 4832\r\nParentImage: C:\\Users\\AtomicRed\\Downloads\\shadow2\\ShadowSteal.exe\r\nParentCommandLine: ShadowSteal.exe\"","version":"5","systemTime":"2021-07-21T18:52:51.7297548Z","eventRecordID":"296414","threadID":"3460","computer":"hrmanager.ExchangeTest.com","task":"1","processID":"2428","severityValue":"INFORMATION","providerName":"Microsoft-Windows-Sysmon"}}} -->
```

Comment:
Maybe not now but going further with this limitation of writing UT, I'm thinking to move this sample traces to the INI file as well, even if they are not UTs. Maybe this helps to make cleaner rule files. What do you think?

Reply:
Agree, not sure if there is a way to make the .ini file to be ignored when the CI runs tests. Perhaps we can use other folder or something similar to avoid execution